recipes/amplisim/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: amplisim
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/rki-mf1/amplisim
+  git_tag: v{{ version }}
+  sha256: 6697558611dfb8b83f77ebc825f83112e9696eea9218dd35e9dac7705bf9f474
```

Comment:
typically we use the export tarball, is there some reason why you need to pull from git?